generated/.tailcallrc.schema.json

```diff
@@ -833,18 +833,6 @@
       "title": "JSON",
       "description": "Field whose value conforms to the standard JSON format as specified in RFC 8259 (https://datatracker.ietf.org/doc/html/rfc8259)."
     },
-    "Key": {
```

Comment:
removed the separate key from the Config because it's generated automatically and otherwise it's possible to define it through common `directives`